src/plugins/workspace/common/constants.ts

```diff
@@ -79,3 +79,94 @@ export const WORKSPACE_APP_CATEGORIES: Record<string, AppCategory> = Object.free
     order: 14000,
   },
 });
+
+export const WORKSPACE_USE_CASES = Object.freeze({
```

Comment:
Shall we add a comment to mention this is a temporary solution, and also briefly describe the future plan?